docker/docker-compose.yml

```diff
@@ -5,7 +5,6 @@
 #   Destroy:        docker compose -f docker-compose.yml -f ./dev/docker-compose.dev.yml down -v --remove-orphans
 
 name: supabase
-version: "3.8"
```

Comment:
Is this breaking anything?

Reply:
Not sure if it'll break things I've removed it because docker was giving me warnings about the fact that it's deprecated. But maybe I'll leave it on for older docker version ?

Reply:
Thanks for context. Removing sounds good then.